### PR TITLE
Fix Grammar who's -> whose

### PR DESCRIPTION
**Pull request by tomrittervg**

### CODE REVIEW

File: spec/1.1/index.bs [hunk 1]
```diff
@@ -711,7 +711,7 @@ Return the {{VRDisplay/displayId}} of the {{VRDisplay}} this {{Gamepad}} is asso
 
 While not directly affecting the API interface and Web IDL, WebVR implementations should maintain the user's expectations of privacy, security, and comfort on the Web by adhering to the following guidelines:
 
-* Trusted UI must be drawn by an independent rendering context who's state is isolated from that of the WebGL contexts provided as a {{VRLayerInit}} source to a {{VRDisplay}}.
+* Trusted UI must be drawn by an independent rendering context whose state is isolated from that of the WebGL contexts provided as a {{VRLayerInit}} source to a {{VRDisplay}}.
```
Comment (cvan) [inline spec/1.1/index.bs]: good catch. also, I found these occurrences too:

- https://github.com/w3c/webvr/blob/master/spec/latest/index.bs#L103
- https://github.com/w3c/webvr/blob/master/spec/latest/index.bs#L303
- https://github.com/w3c/webvr/commit/1e99be61#diff-817a23c792520ce222813d3eff70d031R301

Comment (cvan) [inline spec/1.1/index.bs]: I can merge this and fix the other ones. I figured I'd ask first if you wanted to do it, before I went ahead and merged it though.

Comment (tomrittervg) [inline spec/1.1/index.bs]: Okay done.

Comment (cvan) [inline spec/1.1/index.bs]: it looks like the other changes went missing - can you check again? (sorry for the hassle)

Comment (tomrittervg) [inline spec/1.1/index.bs]: I see three commits attached to this PR now; not sure why you don't...

Comment (toji) [inline spec/1.1/index.bs]: I see all three commits.

**Comment by toji** [comment]
_Sigh._ Thanks for cleaning up after my spotty grammar. I've apparently got some bad muscle memory in this particular case. :P

File: spec/1.1/index.bs (hunk 1 above)
Comment (cvan) [inline spec/1.1/index.bs]: weird, it shows for me fine now. anyway, thanks for making the changes! 👍

**Pull request merged**